src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/SourceMappingFile.cs

```diff
@@ -19,6 +19,8 @@ public class SourceMappingFile
 
     public string? PatchesPath { get; set; }
 
+    public string? SourceMappings { get; set; }
```

Comment:
Could I ask you to add XML docs to properties in this class?

You can just copy them from https://github.com/dotnet/dotnet/blob/main/src/source-mappings.json

Reply:
sure